### DISCUSSION TITLE
Let me tell about place you want to visit!

### DISCUSSION THREAD

**Discussion by honamikoyama** [NAIST-SE/AttractingContributors#1182] [2024 Attraction Contributions]
Do you like traveling? I like traveling. 
Please share your travel interests and travel memories!!

I made a list of places I would like to visit . 
https://github.com/honamikoyama/Travel.git

**Comment by marshr00m** [comment]
I already submitted a pull request. I would appreciate any additions to my list!
https://github.com/marshr00m/AttractingContributors-Where-would-you-like-to-travel

**Comment by LikkaLiu** [comment]
I already submitted a pull request. 
I would appreciate any additions to my list : )
https://github.com/LikkaLiu/AttractingContributors
